src/current/v25.3/ldap-authentication.md

```diff
@@ -79,35 +88,57 @@ If you also intend to configure LDAP Authorization, you will need to include an
 
 If, for LDAPS, you are using a certificate signed by a custom Certificate Authority (CA) that is not in the system's trusted CA store, you will need to configure the CA certificate. This step is only necessary when using certificates signed by your organization's private CA or other untrusted CA.
 
-Set the custom CA certificate:
+**Set the custom CA certificate:**
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
-SET CLUSTER SETTING server.ldap_authentication.domain_ca = '<PEM_ENCODED_CA_CERT>';
+SET CLUSTER SETTING server.ldap_authentication.domain.custom_ca = '<PEM_ENCODED_CA_CERT>';
 ~~~
 
-Configure a client certificate for mTLS if required:
+**Configure a client certificate for mTLS if required:**
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
 SET CLUSTER SETTING server.ldap_authentication.client.tls_certificate = '<PEM_ENCODED_CERT>';
 SET CLUSTER SETTING server.ldap_authentication.client.tls_key = '<PEM_ENCODED_KEY>';
 ~~~
 
-### Step 4: Sync database users
+### Step 4: Configure user creation
 
-{{site.data.alerts.callout_info}}
-LDAP authentication cannot be used for the `root` user or other [reserved identities]({% link {{ page.version.version }}/security-reference/authorization.md %}#reserved-identities). Credentials for `root` must be managed separately using password authentication to ensure continuous administrative access regardless of LDAP availability.
-{{site.data.alerts.end}}
+CockroachDB supports two approaches for the creation of users who will authenticate via LDAP:
 
-Before LDAP authentication can be used for a user, the username must be created directly in CockroachDB. You will need to establish an automated method for keeping users in sync with the directory server, creating and dropping them as needed.
+#### Option 1: Automatic user provisioning (recommended)
 
-For Active Directory deployments, the CockroachDB username can typically be set to match the `sAMAccountName` field from the `user` object. This field name would need to be specified in the HBA configuration using `ldapsearchattribute=sAMAccountName`.
+{% include_cached new-in.html version="v25.3" %}
+
```

Comment:
Maybe no line break here? I think the version tag is formatted **New in v25.3:** so the sentence should follow directly.